solr/schema.xml

```diff
@@ -261,6 +261,7 @@ schema. In this case the version should be set to the next CKAN version number.
     <field name="publishers" type="textgen" indexed="true" stored="true" multiValued="true"/>
     <field name="identifier" type="string" indexed="true" stored="true"/>
     <field name="see_alsos" type="string" indexed="true" stored="true" multiValued="true"/>
+    <field name="private" type="boolean" indexed="true" stored="true" multiValued="false"/>
```

Comment:
Is this line added correctly?

Reply:
Thanks for pointing that out! Yes, it's correct. It's the field we need so we can facet on private/published datasets (e.g. draft/published).